CRM/Upgrade/Incremental/sql/5.63.alpha1.mysql.tpl

```diff
@@ -14,3 +14,11 @@ VALUES
     ( {$domainID}, 'https://civicrm.org/help?src=iam',       '{ts escape="sql" skip="true"}Get Help{/ts}',   'Get Help',   NULL, 'AND', @adminHelplastID, '1', NULL, 2 );
 
 UPDATE IGNORE `civicrm_navigation` SET `name` = 'Register Your Site', `label` = '{ts escape="sql" skip="true"}Register Your Site{/ts}' WHERE `name` = 'Register your site';
+
+{if $multilingual}
+    {foreach from=$locales item=locale}
+        UPDATE IGNORE civicrm_uf_field SET label_{$locale} = '{ts escape="sql"}Contribution Date{/ts}' WHERE field_name = 'receive_date' AND label_{$locale} = '{ts escape="sql"}Received{/ts}';
```

Comment:
This is tricky because the the default label for this was Date Received when added to a profile manually, but the stock reserved profile used Received. So if someone had changed their own profile from Date Received to Received because that's the wording they wanted, then this would update that.

```suggestion
        UPDATE IGNORE civicrm_uf_field SET label_{$locale} = '{ts escape="sql"}Contribution Date{/ts}' WHERE (field_name = 'receive_date' AND label_{$locale} = '{ts escape="sql"}Received{/ts}' AND uf_group_id IN (10,11)) OR (field_name = 'receive_date' AND label_{$locale} = '{ts escape="sql"}Date Received{/ts}' AND uf_group_id NOT IN (10,11));
```
Or something like that. If it's clearer it could be two sql statements.